src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java

```diff
@@ -8,6 +8,7 @@
 
     private Boolean addNewCardEnabled;
     private Boolean validateNewCard;
+    private Boolean debitCardAllowed;
```

Comment:
this field can lead to a fault in hash computation. should be added to toString() method.

Reply:
done